src/frameworks/svelte.json

```diff
@@ -8,13 +8,13 @@
     "configFiles": []
   },
   "dev": {
-    "command": "npm run dev",
-    "port": 5000,
+    "command": "vite dev",
+    "port": 5173,
```

Comment:
I could be wrong here and so this is not a review.

I don't know that this will actually fix #844. The assumption seems to be that a [\[svelte\] dependency](https://github.com/netlify/framework-info/pull/846/files#diff-6407f2d5cb5783d62d23913e41ceaef41eb09d9b0418cf901184b9d88ba72193L6) implies that `svelte-kit` is being used (it may not be), 

```json
{
  "detect": {
    "npmDependencies": ["svelte"]
  }
}
```

and so if `svelte-kit` is being used, so then `vite` must also be (I have not used svelte-kit, but I suspect that other bundlers / task runners are possible?) and that a dev server is being run via that particular bundler / task runner.

Could it be confusing that this filename is `svelte.json`?  Would it make more sense to name it `svelte-kit.json` since that seems to be the basic assumption here?

Reply:
SvelteKit does require vite and uses it for the dev and build commands. Svelte (vanilla) recommends the usage of the Vite template which also uses Vite for the dev and build commands. I think this is the best option going forward to detect the framework correctly and if there is a custom setup, it will need to be manually configured in the `netlify.toml` file.

Reply:
Yes, these values are meant to be the defaults for a framework.

Reply:
Awesome. All is clear, then!

👍🏻